libraries/disp3D/engine/view/orbitalcameracontroller.h

```diff
@@ -107,6 +115,13 @@ class DISP3DSHARED_EXPORT OrbitalCameraController : public Qt3DExtras::QAbstract
      */
     void invertCameraRotation(bool newStatusFlag);
 
+    //=========================================================================================================
+    /**
+     * Sets the angle of the camera for rotating around
+     */
+    void setRotating(int count);     /* The counter for how long the rotation has been happening. */
+    int rotating() const;
```

Comment:
please document each function and their functions values (input and return)

Reply:
I forgot that one! I wasn't sure when I was trying things if they would work so I didn't document but I forgot to go back and fix it